imageroot/actions/get-configuration/20read

```diff
@@ -19,11 +19,22 @@
 # You should have received a copy of the GNU General Public License
 # along with NethServer.  If not, see COPYING.
 #
-
+import os
 import json
 import sys
 import agent
 
+# Prepare return variable
 config = {}
 
+# Read current configuration from Redis
+env = f'module/{os.environ["MODULE_ID"]}/environment'
+rdb = agent.redis_connect()
+
+config["admin_password"] =  rdb.hget(env, "ADMIN_PASSWORD");
```

Comment:
Just an idea: you get the value directly from the ENV without accessing redis at all

Reply:
done